apps/server/src/modules/learnroom/controller/dto/card/card.response.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { DecodeHtmlEntities } from '@shared/controller';
+import { VisibilitySettingsResponse } from './visibility-settings.response';
+
+export class ContentElementResponse {}
+
+type BoardCardType = 'task' | 'content' | 'legacy-task' | 'legacy-lesson';
+
+export class CardResponse {
+	constructor({ id, height, title, elements, cardType, visibilitySettings }: CardResponse) {
+		this.id = id;
+		this.height = height;
+		this.title = title;
+		this.elements = elements;
+		this.cardType = cardType;
+		this.visibilitySettings = visibilitySettings;
+	}
+
+	@ApiProperty({
+		pattern: '[a-f0-9]{24}',
+	})
+	id: string;
+
+	@ApiProperty({
+		description:
+			'The approximate height of the referenced card. Intended to be used for prerendering purposes. Note, that different devices can lead to this value not being precise',
+	})
+	height: number;
+
+	@ApiProperty()
+	@DecodeHtmlEntities()
+	title: string;
```

Comment:
title should not be mandatory - I can imagine cards with only a picture, or something like that, no title
In the case of task-card, the title is considered also a card element, but in the logic we make it mandatory, single one etc